rubin_sim/maf/metrics/brownDwarfMetric.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import healpy as hp
+from .baseMetric import BaseMetric
+import rubin_sim.maf.utils as mafUtils
+from scipy import interpolate
+
+
+class BDParallaxMetric(BaseMetric):
+    """Calculate the distance to which one could reach a parallax SNR for a given object
+    Modification of ParallaxMetric. Taken from:
+    https://github.com/jgizis/LSST-BD-Cadence/blob/main/bd_allLT_baseline_17.ipynb
+
+    Uses columns ra_pi_amp and dec_pi_amp, calculated by the ParallaxFactorStacker.
+
+    Parameters
+    ----------
+    metricName : str, opt
```

Comment:
For doc string parameters, the doc build works better with:

metricName : `str`, opt 

(etc. -- the backticks make the thing be a linkable object in different font. See https://developer.lsst.io/python/numpydoc.html#py-docstring-sections
I still have problems getting the np.ndarray ones to resolve properly).